client/jetpack-cloud/style.scss

```diff
@@ -49,6 +49,35 @@
 	.my-sites__navigation {
 		height: 100%;
 	}
+
+	.site-selector {
+		top: 80px;
+
+		&__sites {
```

Comment:
Can we keep it `.site-selector__sites`. It becomes harder to search by the class name